runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -315,6 +315,21 @@ public void handleJavaScriptResult(String jsonResult) {
         mContentViewCore.getWebContents().evaluateJavaScript(script, coreCallback);
     }
 
+    public void evaluateJavascriptForTests(String script, ValueCallback<String> callback) {
+        if (mNativeContent == 0) return;
+        final ValueCallback<String>  fCallback = callback;
+        JavaScriptCallback coreCallback = null;
+        if (fCallback != null) {
+            coreCallback = new JavaScriptCallback() {
+                @Override
+                public void handleJavaScriptResult(String jsonResult) {
+                    fCallback.onReceiveValue(jsonResult);
+                }
+            };
+        }
+        mContentViewCore.getWebContents().evaluateJavaScriptForTests(script, coreCallback);
+    }
```

Comment:
Because there are only a few duplicated code and only two methods are sharing them, I think there is no need to add a private member method specially.

I added comment in `XWalkViewInternal`.